content/en/docs/reference/using-api/deprecation-policy.md

```diff
@@ -428,6 +428,49 @@ transitions a lifecycle stage as follows. Feature gates must function for no les
 is deprecated it must be documented in both in the release notes and the corresponding CLI help.
 Both warnings and documentation must indicate whether a feature gate is non-operational.**
 
+## Deprecating a metric
+
+Each component of the Kubernetes control-plane exposes metrics (usually the
+`/metrics` endpoint), which are typically ingested by cluster administrators.
+Not all metrics are the same: some metrics are commonly used as SLIs or used
+to determine SLOs, these tend to have greater import. Other metrics are more
+experimental in nature or are used primarily in the Kubernetes development
+process.
+
+Accordingly, metrics fall under two stability classes (`ALPHA` and `STABLE`);
+this impacts removal of a metric during a Kubernetes release. These classes
+are determined by the perceived importance of the metric. The rules for
+deprecating and removing a metric are as follows:
+
+**Rule #9: ALPHA metrics have no guarantees and can be removed at any time.**
+
+**Rule #10: STABLE metrics must undergo a deprecation lifecycle prior to removal.**
+
+   * **STABLE metric: 3 releases**
+   * **STABLE (but deprecated): 2 releases**
+   * **STABLE (but deprecated and now hidden by default): 1 release**
+
+Deprecated metrics have the same stability guarantees of their counterparts. If a stable
+metric is deprecated, then a deprecated stable metric is guaranteed to not change. When
+deprecating a stable metric, a future Kubernetes release is specified as the point from
+which the metric will be considered deprecated.
+
+Deprecated metrics will have their description text prefixed with a deprecation notice
+string '(Deprecated from x.y)' and a warning log will be emitted during metric
```

Comment:
What is the point to this warning log? consumers of the metrics aren't going to be looking at the logs at that time.

Reply:
I disagree actually. Cluster admins also tend to also be consumers of metrics, so they may end up looking at the logs.

Reply:
Sure, I don't object to logging them, but I can't believe it's going to happen very often that the person in charge of metrics ingestion is going to happen to be reading the logs and learn about the deprecation there for the first time, in advance of their metrics ingestion breaking.